apps/address-service/domains/address/access/Address.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Generated by `createschema address.Address 'source:Text; address:Text; key:Text; meta:Json'`
+ */
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+async function canReadAddresses ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin || user.isSupport) return {}
+
+    return false
+}
+
+async function canManageAddresses ({ authentication: { item: user }, originalInput, operation, itemId }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    return !!user.isAdmin
```

Comment:
What about supports?

Reply:
These models (`Address` and `AddressSource`) will be created with external data, so we restrict editing them.